cli/template/base/package.json

```diff
@@ -14,22 +14,25 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
-    "@t3-oss/env-nextjs": "^0.10.1",
-    "geist": "^1.3.0",
-    "next": "^15.0.1",
-    "react": "^18.3.1",
```

Comment:
I think we should stay on 18 until it's properly out. We're just gonna get a bunch of issues about peer dep warnings otherwise

Reply:
> I think we should stay on 18 until it's properly out. We're just gonna get a bunch of issues about peer dep warnings otherwise

Isn't React 19 the minimum requirement for Next 15?
https://nextjs.org/docs/app/building-your-application/upgrading/version-15#react-19
> The minimum versions of react and react-dom is now 19.

Reply:
I think they walked back on that, interesting that it's in the release notes though..

Peer dep allows it and the cli today uses Next15 w/ React 18 👀👀

Reply:
Ah that's for pages router. App router uses R19 no matter what you have in deps I guess https://nextjs.org/blog/next-15#react-19

Reply:
Makes sense